app/upgrades/v1/bank_send_list.go

```diff
@@ -0,0 +1,65 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func BankSendList(ctx sdk.Context) (ResultList, error) {
+	ctx.Logger().Info(fmt.Sprintf("bank send list:%s", UpgradeName))
+
+	// Read file and get list
+	var result ResultList
+	json.Unmarshal([]byte(BANK_SEND_LIST), &result)
+
+	return result, nil
+}
+
+const BANK_SEND_LIST string = `{
+	"validator": [
+		{
+			"fromAddress": "ununifi132ap8qzhmzn9edyjzz290xvr96dgzp2khhapk7",
+			"toAddress": "ununifi132ap8qzhmzn9edyjzz290xvr96dgzp2khhapk7",
+			"amount": "uguu",
+			"denom": 100001,
```

Comment:
switch the value in "amont" and "denom".
"denom" means kinda ticker of the token, which in this context is "uguu".

Reply:
🙇‍♂️